CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 
 ### Enhancements
 
+- Add liveness probe to `keepalived` pod 
+  (PR[#4118](https://github.com/scality/metalk8s/pull/4118))
```

Comment:
Nit: Can you just move this at the end of `Enhancements` list :)

Reply:
Oopsie